### DISCUSSION TITLE
How to push data into GPU?

### DISCUSSION THREAD

**Discussion by hahahaha90** [pyg-team/pytorch_geometric#5382] [Q&A]
Hi, everyone. 
I just want to speed up data load by pushing all the data into GPU first, then train them in mini-batch. How can I organize my code? 
Thanks.

**Comment by anthonysirico** [comment]
I just had that answered, try looking at this post.
https://github.com/pyg-team/pytorch_geometric/discussions/5389

**Reply by hahahaha90** [reply]
thanks
